dnssec_keyscan.go

```diff
@@ -46,15 +51,15 @@ func (k *DNSKEY) ReadPrivateKey(q io.Reader, file string) (PrivateKey, error) {
 		}
 		priv.PublicKey = *pub
 		return (*DSAPrivateKey)(priv), e
-	case "1 (RSAMD5)":
+	case RSAMD5:
 		fallthrough
-	case "5 (RSASHA1)":
+	case RSASHA1:
 		fallthrough
-	case "7 (RSASHA1NSEC3SHA1)":
```

Comment:
[ Quoting <notifications@github.com> in "Re: [dns] Use algorithm number to d..." ]
>>  		fallthrough
>> -	case "7 (RSASHA1NSEC3SHA1)":
>
>The `RSASHA1NSECSHA1` was clearly a typo and was supposed to be `RSA-SHA1-NSEC-SHA1`. Could've just fixed the typo and be done with it. However, as we discovered, BIND9 uses completely different human readable algorithm names (ie instead of `RSA-SHA1-NSEC-SHA1` it uses `NSEC3RSASHA1`, instead of `RSAMD5 ` it uses `RSA`, etc).
>
>BIND9 also doesn't care about the human readable version of the algorithm when it's parsing the files (at least if I read the code correctly). So in order to fix reading of private keys generated by this library and BIND9 we should also ignore the human readable version of the algorithm. Which this PR does.
>
>*Note: BIND9 should've been able to read private keys generated by this lib regardless of this PR since it didn't care about the human readable names of algos.*

Ah, sorry. We only looking at the algorithm number *only* in this PR. Which
should solve this.